### ISSUE TITLE
Add `"noopener noreferrer"` rel attribute to external links in the footer.

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#4750]
## Problem Statement

For added security, it is recommended to include a `rel="noopener noreferrer"` attribute to all external links. In the footer, on all pages, the links to Facebook, Instagram, Twitter, Flickr and YouTube. For reference, please see https://pointjupiter.com/what-noopener-noreferrer-nofollow-explained/

Note: This should also include all external links on this page: https://www.va.gov/opa/socialmedia.asp (will likely need a separate ticket)

## Goal

Increase security of the site

## Objectives or Key Results this is meant to further

Increase security/prevent phishing attacks

## Acceptance Criteria
- [ ] All external links in the footer include a `rel="noopener noreferrer"` attribute

**Comment by AkucherGCIO** [comment]
2020-26-02 PW Planning
Footer work has been handled by the Public Websites team.

@brandonrapp - Nick will be working on this issue.

**Comment by ncksllvn** [comment]
This is a _little_ more complicated than I expected so I'm going to raise my original estimate to a 3 before starting. LMK if we shouldn't be bumping estimates after grooming/planning through @AkucherGCIO

**Comment by ncksllvn** [comment]
Merged, will go live today -

![image](https://user-images.githubusercontent.com/1915775/75788033-a28cae80-5d35-11ea-9b6a-fb118560e5c2.png)

**Comment by AkucherGCIO** [comment]
@ncksllvn  - Regarding changing estimate after planning, it is ideal to estimate correctly before Sprint starts.  But after the Planning, if estimate changes significantly then that's an issue as it might affect the other tasks taken in the Sprint.  Hope that makes sense.

**Comment by ncksllvn** [comment]
This is live! It doesn't have a visual effect so it's hard to verify. But if you inspect the Facebook link in the footer, you should see `rel="noopener noreferrer"`.

**Comment by AkucherGCIO** [comment]
03/06/20 PW Standup
@kelsonic - Please validate this ticket.

**Comment by AkucherGCIO** [comment]
Validated and closing this ticket.
